extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibConfig.java

```diff
@@ -29,7 +29,7 @@ public class JibConfig {
      * "registry.access.redhat.com/ubi8/ubi-minimal" which is a bigger base image, but provide more built-in utilities
      * such as the microdnf package manager.
      */
-    @ConfigItem(defaultValue = "quay.io/quarkus/quarkus-micro-image:1.0")
+    @ConfigItem(defaultValue = "quay.io/cescoffi/quarkus-micro:2.0")
```

Comment:
Should the `-image*-` really be removed here? You keep it everywhere else?

Reply:
No no, my mistake, I will fix the name. 

Be aware that it's just tests, far from being ready. I test images on both architectures.
I may have to introduce a `platform` attribute in Quarkus to ask ARM machine to build AMD images (and this is not going to be fun as I would need to change the generated docker file, jib ....)